src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/LicenseScanTests.cs

```diff
@@ -188,7 +188,7 @@ private void FilterFiles(ScancodeResults scancodeResults)
         // In other words, the baseline will be fully representative of the licenses that apply to the files that are listed there.
 
         // We only care about the license expressions that are in the target repo.
-        ExclusionsHelper exclusionsHelper = new("LicenseExclusions.txt", BaselineSubDir, _targetRepo);
+        ExclusionsHelper exclusionsHelper = new("LicenseExclusions.txt", BaselineSubDir, _relativeRepoPath);
```

Comment:
How about updating this so that the regex matches only on the beginning of the string being tested. That avoids matching msbuild on something like `src/sdk/src/msbuild`. Or does the string not start with `src/<repo>`? Is it an absolute path? Either way, we should define the regex to match with content that starts from the beginning of the string to avoid false matches.

Reply:
That's a good idea. All of the exclusions should start with `src/<repo>`, I've updated this accordingly in https://github.com/dotnet/sdk/pull/41028/commits/d3bcab96be4de1452d90ae60a51a0f6821de02cb